Composer/packages/client/src/pages/design/createDialogModal.tsx

```diff
@@ -43,37 +43,35 @@ export const CreateDialogModal: React.FC<CreateDialogModalProps> = props => {
     });
   };
 
-  const nameRegex = /^[a-zA-Z0-9-_.]+$/;
-  const validateForm = (data: DialogFormData) => {
-    const errors: { name?: string } = {};
-    const { name } = data;
-
+  const _onGetErrorMessage = (name: string) => {
     if (name) {
       if (!nameRegex.test(name)) {
```

Comment:
This is used before its defined.

Reply:
It's defined on line 46 and used on line 88. I don't see the issue.